mysql-test/lib/My/Debugger.pm

```diff
@@ -144,7 +144,10 @@ sub do_args($$$$$) {
   my $v = $debuggers{$k};
 
   # on windows mtr args are quoted (for system), otherwise not (for exec)
-  sub quote($) { $_[0] =~ /[; ]/ ? "\"$_[0]\"" : $_[0] }
+  sub quote($) {
+    my $tmp= $_[0];
```

Comment:
Hi @vuvova  have updated regex and tried to remove quotes from parameter like `"--replicate-rewrite-db=t->x"` before quoting. I had to make local copy because when I tried to use regex on `$_[0]` I ended with `Modification of a read-only value attempted at lib/My/Debugger.pm line 151.`